assets/templates/status.hbs

```diff
@@ -4,24 +4,50 @@
 		<meta charset="UTF-8">
 		<title>Tailor Status</title>
 		<style>
-			#statuses {
-				margin: auto;
-				background-color: #FAFAFA;
-				width: fit-content;
-				min-width: 500px;
-				padding: 1em;
-				border-radius: 0.5em;
-				border-style: solid;
-				border-width: 1px;
-				border-color: #D0D0D0;
-				box-shadow: 0 4px 8px rgba(0, 0, 0, 0.2), 0 6px 20px rgba(0, 0, 0, 0.19)
-			}
-		</style>
+            body {
+                font-family: -apple-system, BlinkMacSystemFont, "Segoe UI",
+                    Helvetica, Arial, sans-serif, "Apple Color Emoji",
+                    "Segoe UI Emoji", "Segoe UI Symbol";
+                font-size: 18px;
+                margin: 0;
+                padding: 0;
+            }
+            h1 {
+                background-color: #24292e;
+                color: rgba(255,255,255,0.75);
+                font-size: 30px;
+                margin: 0 auto;
+                max-width: 980px;
```

Comment:
This looks better without the `max-width` on my screen

Reply:
Nevermind, when I used inspection I used id instead of class for the header definition